app/models/concerns/mods_solr_document.rb

```diff
@@ -0,0 +1,162 @@
+# frozen_string_literal: true
+
+# build mods xml document: see https://www.loc.gov/standards/mods/mods-outline-3-5.html
+# TODO: load terms dynamically via AllinsonFlex
+module ModsSolrDocument
+  extend ActiveSupport::Concern
+
+  def to_oai_mods
```

Comment:
I'm going to guess that Rubocop will not like this method length

Reply:
Probably not.